Evaluation/Input/evaluation_plan.json

```diff
@@ -123,24 +123,27 @@
             {
                 "Project": "Compound",
                 "Simulation": "Sim 1 - intravenous",
-                "SectionId": 33
+                "SectionReference": "ct-profiles"
```

Comment:
Using the new concept of SectionReference instead of SectionId.
At the moment both are supported (in order to keep current quali plans runnable) - but the support will be dropped one day for sure